x-pack/plugins/endpoint/public/plugin.ts

```diff
@@ -24,8 +25,19 @@ export class EndpointPlugin
       EndpointPluginSetupDependencies,
       EndpointPluginStartDependencies
     > {
-  public setup(_core: CoreSetup, plugins: EndpointPluginSetupDependencies) {
+  public setup(core: CoreSetup, plugins: EndpointPluginSetupDependencies) {
     const resolverEmbeddableFactory = new ResolverEmbeddableFactory();
+    core.application.register({
+      id: 'endpoint',
+      title: i18n.translate('xpack.endpoint.pluginTitle', {
+        defaultMessage: 'Endpoint',
+      }),
+      async mount(context, params) {
```

Comment:
`context` has been deprecated. See this PR on how to get access to Core services: https://github.com/elastic/kibana/pull/50231